ios/FluentUI/Tab Bar/TabBarItemView.swift

```diff
@@ -29,16 +29,13 @@ class TabBarItemView: UIControl, TokenizedControlInternal {
 
     override var isEnabled: Bool {
         didSet {
-            titleLabel.isEnabled = isEnabled
-            imageView.tintAdjustmentMode = isEnabled ? .automatic : .dimmed
             isUserInteractionEnabled = isEnabled
+            updateColors()
```

Comment:
is this ok? updateColors() doesn't change titlelabel.isEnabled?

Reply:
That'll be addressed in a later PR to freshen up the stale tab bar.